guides/serialization_and_saving.py

```diff
@@ -177,26 +181,52 @@ def call(self, inputs):
             x = layer(x)
         return x
 
+    def get_config(self):
+      return {"hidden_units": self.hidden_units}
+
+    @classmethod
+    def from_config(cls, config):
+      return cls(**config)
 
 model = CustomModel([16, 16, 10])
 # Build the model by calling it
 input_arr = tf.random.uniform((1, 5))
 outputs = model(input_arr)
 model.save("my_model")
 
+# Option 1: Load with the custom_object argument.
+loaded_1 = keras.models.load_model(
+    "my_model", custom_objects={"CustomModel": CustomModel})
+
+# Option 2: Load without the CustomModel class.
+
 # Delete the custom-defined model class to ensure that the loader does not have
 # access to it.
 del CustomModel
 
-loaded = keras.models.load_model("my_model")
-np.testing.assert_allclose(loaded(input_arr), outputs)
+loaded_2 = keras.models.load_model("my_model")
+np.testing.assert_allclose(loaded_1(input_arr), outputs)
+np.testing.assert_allclose(loaded_2(input_arr), outputs)
 
 print("Original model:", model)
-print("Loaded model:", loaded)
+print("Model Loaded with custom objects:", loaded_1)
+print("Model loaded without the custom object class:", loaded_2)
+
+
+"""
+The first loaded model is loaded using the config and `CustomModel` class. The second
+model is loaded by dynamically creating the model class that acts like the original model.
+"""
+
 
 """
-As seen in the example above, the loader dynamically creates a new model class
-that acts like the original model.
+#### Configuring the SavedModel
+
+*New in TensoFlow 2.4*
```

Comment:
Please update this to reflect the final API

Reply:
Done